tarpc/src/transport/tcp.rs

```diff
@@ -0,0 +1,62 @@
+use std::io;
+use std::net::{SocketAddr, TcpListener, TcpStream, ToSocketAddrs};
+use std::time::Duration;
+
+/// A transport for TCP.
+pub struct TcpTransport<A: ToSocketAddrs>(pub A);
+impl<A: ToSocketAddrs> super::Transport for TcpTransport<A> {
```

Comment:
I'll try but it might be from cargo fmt.
On Mar 2, 2016 9:29 PM, "shaladdle" <notifications@github.com> wrote:

> In tarpc/src/transport/tcp.rs
> <https://github.com/google/tarpc/pull/27#discussion_r54837589>:
>
> > @@ -0,0 +1,62 @@
> > +use std::io;
> > +use std::net::{SocketAddr, TcpListener, TcpStream, ToSocketAddrs};
> > +use std::time::Duration;
> > +
> > +/// A transport for TCP.
> > +pub struct TcpTransport<A: ToSocketAddrs>(pub A);
> > +impl<A: ToSocketAddrs> super::Transport for TcpTransport<A> {
>
> Add a line in between the struct and the impl?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/tarpc/pull/27/files#r54837589>.
>